terminal.html

```diff
@@ -9,7 +9,6 @@
 <body>
     <div id="terminal"></div>
 </body>
-<script src="js/input_checker.js"></script>
```

Comment:
Does the terminal not need to use input_checker anymore?

Reply:
It does. But it's now created in the parent (mainview), so this line need not be here.